app/assets/stylesheets/application.scss

```diff
@@ -20,4 +20,17 @@
 
 .autocomplete__wrapper {
   width: 66.66% !important;
+}
+
+a[href="none"] {
```

Comment:
required for the "no" view only pages - devs will be having a chat tomorrow to discuss whether we take this solution or an alternative forward